### ISSUE TITLE
parallel -B for one distro only

### ISSUE THREAD

**Issue by QuLogic** [juhp/fbrnch#15]
I have some packages where others have taken up EPEL packaging. I do not bother with those, nor update them. However, `fbrnch parallel -B` attempts to checkout and build those branches. It would be nice to have a global setting of some sort, or different flag, so that branches for only one distro would count in parallel. I would only want to do `f??` branches, but an epel packager might want `epel?` branches.

**Comment by juhp** [comment]
This is a good suggestion, thank you, and should not be too hard to implement.

**Comment by juhp** [comment]
A simpler solution might be to make -B only imply Fedora (ie not EPEL).

**Comment by juhp** [comment]
A workaround of course would be to do `fbranch parallel -x epel8` - that's a bit more typing of course.

**Comment by juhp** [comment]
It took a while, but finally I implemented `[--all-fedora|-F]` (and `--all-epel`).
It should be in 0.6.5 builds soon.
